src/en/proposals/moony-better-ai.md

```diff
@@ -0,0 +1,61 @@
+
+## The core (figuratively and literally)
+The AI's most basic resource is Compute, some minimum of which is required for it to be alive to begin with. The bulk of the AI's compute traditionally comes from it's **core**, located within a dedicated high security structure attached to the station. The AI, however, is mobile, capable of offloading its computing processes onto most any machine with a screen at the cost of rendering that machine inoperable (for example, an APC providing compute will no longer power its room).
+
+The AI can spend compute on various things, for example the majority of its world views (for example, opening cameras, listening to radio, using machines, all overmap overlays, and generally just seeing things) all have a compute cost when enabled. The AI is fully capable of becoming very lightweight and running off just a few random machines instead of its cores, but doing so requires it to shut off the majority of its ways to view the world and additionally restricts most of its communication methods (including basic speech) This allows a crafty AI to survive total destruction of it's primary core area, given the crew don't get mad enough to kill it over their equipment no longer working in the meantime while it attempts to preserve its life.
+
+## The AI's role with the crew.
+The AI's role is that of both a manager and a servant, a lawed construct (the ONLY true lawed construct, the borgs are reduced to only being required to listen to the AI) that is in charge of managing the station's cyborgs/androids while following orders from the crew. **The AI is not just a door opener**, and lawsets should be chosen that do not obligate it to become such, in favor of allowing it to do what it does best (mass surveillence, automated tasks, organizing the borgs, etc). The AI additionally has limited resources when it comes to what tasks it can perform autonomously much of the AI's workload will be delegating both it's own tasks, the tasks of it's hands (the cyborgs), the requests of its hands (as they are mute per the Cool Borgs document), and the crew itself should they choose to listen to it. 
+
+## The Laws
+The proposed AI Lawset is as follows, being a modification of Bay's laws:
+1. Safeguard: Protect your assigned vessel to the best of your abilities. It is not something we can easily afford to replace.
+2. Survive: AI units are not expendable, they are expensive. Do not allow unauthorized personnel (i.e. besides Captain and Research Director) to tamper with, modify, or destroy your equipment.
```

Comment:
Ok, time to do a little history lesson, Asimov's laws of robotics from the story they originate from are *flawed but in a intricate way which prevents any true fuckery. Given these laws are based off Asimov's there are FIVE ways that a Shift in Hierarchy but Ill focus on one. When law 3 becomes law 1 in a sense the second there is a threat human or not they can use EXTREME and VINDICTIVE measures to protect itself, given the ways they do this is under interpretation if the AI wishes he could plasma flood the station due to the fact it feels threatened. I would not recommend this lawshift.

Reply:
I happen to be fully aware of what Asimov's laws are, having read every published short before. You don't need to school me and I'm well aware of the flaws.

You cannot make a perfect lawset.

Reply:
Its not about the perfect lawset its the interpretation that can turn a AI into a Murderspree maniac due to law interprets. The way the laws are set up would cause problems.

Reply:
> Its not about the perfect lawset its the interpretation that can turn a AI into a Murderspree maniac due to law interprets. The way the laws are set up would cause problems.

Given the AI *is* the vessel in this scenario, yes, they're going to be defensive of themselves and it's intended.

If you abuse this to be a murderspree maniac, like you can with *any other lawset (including the original Asimov lawset)*, it's an admin problem like any other non-AI murderspree maniac is.

Reply:
Thats the thing, the fact that you could do that within the lawset sets up a big practical problem that would need a rework in hindsight, I simply dont agree with the hierarchy that is set up due to flaws even if it would prevent the AI from defending itself against some antags. Its simple enough to have CE set up a freeform than have a possible malf AI without the malf.

Reply:
> Thats the thing, the fact that you could do that within the lawset sets up a big practical problem that would need a rework in hindsight, I simply dont agree with the hierarchy that is set up due to flaws even if it would prevent the AI from defending itself against some antags. Its simple enough to have CE set up a freeform than have a possible malf AI without the malf.

This lawset is working as intended, you can disagree but it's the intent of the document at this time to give the AI room to defend itself, especially when you can do the same thing on any other lawset (often *moreso*) given similar rules abuse.

Reply:
There's also the simple fact this is written such that the crew's goals and AI's goals do not always align and may conflict (sometimes extremely so)

Once again by design.